hitrace/src/lib.rs

```diff
@@ -115,6 +132,27 @@ impl ScopedTrace {
             phantom_data: PhantomData,
         }
     }
+
+    /// Starts a count trace for the given name and count value.
+    ///
+    /// This method is used to trace the value change of an integer variable.
+    #[must_use]
+    pub fn start_count_trace<T: AsRef<CStr>>(name: &T, count: i64) -> Self {
```

Comment:
These methods shouldn't be added to `ScopedTrace`. `OH_HiTrace_CountTrace` behaves similarly to Events in [tracing-rs](https://docs.rs/tracing/latest/tracing/), i.e. are a single point in time and don't have an end.

Reply:
moved it out